src/view/workflow/index.js

```diff
@@ -51,85 +44,20 @@ export default View.extend({
     }
     View.prototype.remove.apply(this, arguments)
   },
-  bindings: {
-    warningToggle: [
-      {
-        type: 'booleanClass',
-        name: 'btn-danger',
-        hook: 'warning-indicator'
-      }, {
-        type: 'booleanAttribute',
-        name: 'disabled',
-        hook: 'warning-indicator',
-        invert: true
-      }
-    ],
-    clearBtn: {
-      type: 'toggle',
-      hook: 'clear'
-    }
-  },
-  events: {
-    'click button[data-hook=fit]':'onClickFit',
-    'click button[data-hook=center]':'onClickCenter',
-    'click button[data-hook=redraw]':'onClickRedraw',
-    'click button[data-hook=clear]':'onClickClear',
-    'click button[data-hook=warning-indicator]':'onClickWarningIndicator',
-  },
-  onClickWarningIndicator (event) {
-    event.preventDefault()
-    event.stopPropagation()
-
-    this.trigger('click:warning-indicator')
-
-    return false
-  },
-  onClickFit (event) {
-    event.preventDefault()
-    event.stopPropagation()
-
-    this.cy.fit()
-
-    return false
-  },
-  onClickCenter (event) {
-    event.preventDefault()
-    event.stopPropagation()
-
-    this.cy.center()
-
-    return false
-  },
-  onClickRedraw (event) {
-    event.preventDefault()
-    event.stopPropagation()
-
-    this.updateCytoscape()
-
-    return false
-  },
-  onClickClear (event) {
-    event.preventDefault()
-    event.stopPropagation()
-
-    this.trigger('click:clear')
-
-    return false
-  },
-  updateCytoscape () {
+  updateCytoscape (positions) {
     if (this.cy) {
       this.cy.destroy()
       this.cy = null
     }
 
-    this.renderCytoscape()
+    this.renderCytoscape(positions)
 
     this.cy.center()
     this.cy.fit()
 
     return this
   },
-  renderCytoscape () {
+  renderCytoscape (positions) {
```

Comment:
el render no tiene que llevar parámetros.
tiene que usar this.positions
```suggestion
  renderCytoscape () {
```